configure

```diff
@@ -4,6 +4,10 @@
 # and released into the Public Domain.
 
 Makefile() {
+  if [ $strip -ne 0 ] ; then
```

Comment:
I should probably do some better reorganization of these, but that can be dealt with later.